stdlib/http/src/main/java/org/ballerinalang/net/http/actions/websocketconnector/PushText.java

```diff
@@ -52,8 +54,15 @@ public static Object externPushText(Strand strand, ObjectValue wsConnection, Str
                     .getNativeData(WebSocketConstants.NATIVE_DATA_WEBSOCKET_CONNECTION_INFO);
             ChannelFuture future = connectionInfo.getWebSocketConnection().pushText(text, finalFrame);
             WebSocketUtil.handleWebSocketCallback(callback, future, log);
+            //Observe text message sent
+            WebSocketObservabilityUtil.observeSend(strand, WebSocketObservabilityConstants.MESSAGE_TYPE_TEXT,
+                                                   connectionInfo);
         } catch (Exception e) {
-            log.error("Error occurred when pushing text data", e);
+            //Observe text message send error
+            WebSocketObservabilityUtil.observeError(strand, WebSocketObservabilityUtil.getConnectionInfo(wsConnection),
+                                                    WebSocketObservabilityConstants.ERROR_TYPE_MESSAGE_SENT,
+                                                    WebSocketObservabilityConstants.MESSAGE_TYPE_TEXT,
+                                                    e.getMessage());
```

Comment:
We should note that the value of `e.getMessage` can be null? Are we handling it properly?